django_project/support/urls.py

```diff
@@ -0,0 +1,13 @@
+from django.urls import path, include
+from rest_framework.routers import DefaultRouter
+from .views import TicketViewSet
+from .views import IssueTypeViewSet
+
+
+router = DefaultRouter()
+router.register(r'tickets', TicketViewSet, basename='ticket')
+router.register(r'issue_types', IssueTypeViewSet, basename='issue_type')
+
+urlpatterns = [
+    path('api/', include(router.urls)),
```

Comment:
@tinashechiraya I think this might create conflicts if we add another API URL in a different app. For example, in the alert app, if we want to use 'api/', it won’t work because the URL is already defined here.